.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This file should not be committed. Add it to your global gitignore

Reply:
Thanks, fixed!